cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,49 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import HomePageObject from '../support/pages/home.pageObject';
+import { faker } from '@faker-js/faker';
+
+const homePage = new HomePageObject();
+let mainUser, followUser;
+
 describe('User', () => {
   before(() => {
+    cy.task('db:clear');
+
+    mainUser = {
+      email: faker.internet.email(),
+      username: faker.internet.userName(),
+      password: faker.internet.password(10)
+    };
+
+    followUser = {
+      email: faker.internet.email(),
+      username: faker.internet.userName(),
+      password: faker.internet.password(10)
+    };
+
+    cy.register(mainUser.email, mainUser.username, mainUser.password);
+    cy.register(followUser.email, followUser.username, followUser.password);
```

Comment:
Ensure that the `cy.register` command is correctly implemented and that it verifies the registration process for both `mainUser` and `followUser`. This will help confirm that both users are registered successfully before proceeding with the tests.